EFCore.CheckConstraints/CheckConstraints/Internal/CheckConstraintsOptionsExtension.cs

```diff
@@ -11,17 +11,22 @@ public class CheckConstraintsOptionsExtension : IDbContextOptionsExtension
     {
         private DbContextOptionsExtensionInfo _info;
         private bool _enumCheckConstraintsEnabled;
+        private bool _discriminatorCheckConstraintsEnabled;
```

Comment:
Now this is what I was referring as inconsistency.

Reply:
True.